### ISSUE TITLE
 Narrator and JAWS both are  not announcing the sorting state of already sorted column header of the DetailsList

### ISSUE THREAD

**Issue by v-ajrai** [microsoft/fluentui#16787]
Narrator and JAWS both are  not announcing the sorting state of already sorted column header of the DetailsList

### Environment Information
For narrator: Edge Version 88.0.705.56 (Official build) (64-bit)
For Jaws: Chrome Version 87.0.4280.88 (Official Build) (64-bit)

### Describe the issue:
 Narrator and JAWS both are  not announcing the sorting state of already sorted column header of the DetailsList

### Please provide a reproduction of the issue in a codepen:
https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist
![issue_detail_list](https://user-images.githubusercontent.com/62233982/106732396-aa845b00-6636-11eb-9582-708871406f61.png)

#### Actual behavior:

 Narrator and JAWS both are  not announcing the sorting state of already sorted column header of the DetailsList

#### Expected behavior:
 Narrator and JAWS both should announce the sorting state of already sorted column header of the DetailsList

**Comment by v-ajrai** [comment]
Any ETA on this ?

**Comment by joschect** [comment]
@v-ajrai Thanks for the submission! I'm taking a look at this today.

**Comment by joschect** [comment]
@v-ajrai I've taken a look at this and I can't reproduce this with either narrator or NVDA, I suspect that this is a problem with Jaws, especially considering that we have the appropriate aria-labeling on the elements (see screenshot below)

![image](https://user-images.githubusercontent.com/20801176/109359965-399e2080-783b-11eb-9561-b46a8265cff4.png)

**Comment by jeffersoneagley** [comment]
I keep having similar sorting issues logged against my portals.

**Comment by VasimPatel** [comment]
Hello, I noticed the resolution:external tag, any recommended workaround or new info on this? I am facing the same issue.
